src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -282,7 +300,7 @@ uiModules.get('apps/management')
 
   $scope.$watchMulti([
     'controller.formValues.nameIsPattern',
-    'controller.formValues.nameInterval.name'
+    'controller.formValues.nameInterval.name',
```

Comment:
Some people would shake their heads and tsk at these types of "unrelated" changes, but I applaud you, sir.